app/javascript/chat/compose.jsx

```diff
@@ -31,49 +30,39 @@ export default class Chat extends Component {
       handleEditMessageClose,
       handleMention,
       handleKeyUp,
-      editMessageHtml,
     } = this.props;
 
     return (
-      <div className="messagecomposer">
-        <div className="messageToBeEdited">
-          <div className="message">
-            <span className="editHead">Edit Message</span>
-            <div
-              // eslint-disable-next-line react/no-danger
-              dangerouslySetInnerHTML={{
-                __html: editMessageHtml,
-              }}
-            />
-          </div>
-          <div
-            className="closeEdit"
-            role="button"
-            onClick={handleEditMessageClose}
-            tabIndex="0"
-            onKeyUp={e => {
-              if (e.keyCode === 13) handleEditMessageClose();
-            }}
-          >
-            x
-          </div>
-        </div>
+      <div className="composer-container__edit">
         <textarea
-          className="messagecomposer__input"
+          className="crayons-textfield composer-textarea__edit"
           id="messageform"
           placeholder="Let's connect"
           onKeyDown={handleKeyDownEdit}
           onKeyPress={handleMention}
           onKeyUp={handleKeyUp}
           maxLength="1000"
         />
-        <button
-          type="button"
-          className="messagecomposer__submit cta"
-          onClick={handleSubmitOnClickEdit}
-        >
-          Save
-        </button>
+        <div class="composer-btn-group">
+          <button
+            type="button"
+            className="crayons-btn"
+            onClick={handleSubmitOnClickEdit}
+          >
+            Save
+          </button>
+          <button
+            type="button"
+            className="crayons-btn crayons-btn--secondary"
+            onClick={handleEditMessageClose}
+            tabIndex="0"
+            onKeyUp={e => {
+              if (e.keyCode === 13) handleEditMessageClose();
+            }}
```

Comment:
I saved these lines of code from when it was previously a `div`, but I don't know if they're unnecessary.